compiler/src/lntoamm/Microstatement.ts

```diff
@@ -179,190 +180,35 @@ class Microstatement {
     return original
   }
 
-  // TODO: Eliminate ANTLR
-  static fromVarAst(varAst: any, scope: Scope, microstatements: Array<Microstatement>) {
```

Comment:
I had started to translate this, and then I noticed that absolutely nothing used it. It should've been removed in the statement refactor PR last month.